pkg/client/versioned/typed/workflow/v1/fake/fake_workflow.go

```diff
@@ -0,0 +1,123 @@
+// Generated file, do not modify manually!
```

Comment:
Generally speaking... do we really want to be committing generated files?

Reply:
It's the common practice in kube. Not saying is right: but it's the way it is. At the moment I would commit the generated files. Eventually removing them if we have a strategy for the generator.